client/src/components/Monument/Monument.js

```diff
@@ -83,14 +83,21 @@ export default class Monument extends React.Component {
     capitalize(word) {
         if (word) return word.charAt(0).toUpperCase() + word.slice(1).trim();
         else return word;
-    }
+    };
```

Comment:
```suggestion
    }
```
I think there's semicolons on some of the class functions here because I copied them from another place, you usually don't put semicolons on class functions in JS

Reply:
also I will get rid of those other ones if I ever remember

Reply:
feel free to skip this suggestion for now, super minor

Reply:
I'll remove them all